config/acme/machines/config_batch.xml

```diff
@@ -435,25 +437,25 @@
       <directive>--ntasks-per-node={{ tasks_per_node }}</directive>
     </directives>
     <queues>
-      <queue walltimemin="0" walltimemax="01:00:00" jobmin="0" jobmax="64" default="true">lr3</queue>
+      <queue walltimemax="01:00:00" jobmin="0" jobmax="64" default="true">lr3</queue>
     </queues>
   </batch_system>
 
   <batch_jobs>
     <!-- order matters, with no-batch jobs will be run in the order listed here -->
     <job name="case.run">
       <template>template.case.run</template>
-      <task_count>default</task_count>
+      <node_count>default</node_count>
       <prereq>$BUILD_COMPLETE and not $TEST</prereq>
     </job>
     <job name="case.test">
       <template>template.case.test</template>
-      <task_count>default</task_count>
+      <node_count>default</node_count>
       <prereq>$BUILD_COMPLETE and $TEST</prereq>
     </job>
     <job name="case.st_archive">
       <template>template.st_archive</template>
-      <task_count>1</task_count>
+      <node_count>1</node_count>
```

Comment:
I don't think you want to change this, case.st_archive in particular is a single task job and other jobs you may optionally change here may need the finer control of task count.